testsupport/init.go

```diff
@@ -209,13 +214,17 @@ func WaitForDeployments(t *testing.T) wait.Awaitilities {
 		// Also verify the autoscaling buffer in both members
 		webhookImage := initMemberAwait.GetContainerEnv(t, "MEMBER_OPERATOR_WEBHOOK_IMAGE")
 		require.NotEmpty(t, webhookImage, "The value of the env var MEMBER_OPERATOR_WEBHOOK_IMAGE wasn't found in the deployment of the member operator.")
-		err = initMember2Await.WaitUntilWebhookDeleted(t) // webhook on member2 should be deleted
-		require.NoError(t, err)
+		if isSecondMemberMode {
```

Comment:
btw, you could do 
```suggestion
		if IsSecondMemberMode(t) {
```
without storing it in a variable and passing it through the functions. I know that it would call the `os.Getenv` a few times, but it's fine and it would be easier to read, WDYT?

Reply:
I agree! Tysm!